### PR TITLE
fix: print block strict connection

### LINKED ISSUE #1451: strict-connection-checker demo does not allow the print block to be dragged from the flyout
**Category**

 - Plugins

**Component**

strict-connection-checker

**Describe the bug**

You cannot drag the `print` block from the toolbox, because its input has a `null` check, and its text shadow block has a `String` check.

**To Reproduce**

1. Go to the strict-connection-checker demo: https://google.github.io/blockly-samples/plugins/strict-connection-checker/test/
2. Attempt to drag the print block from the flyout.
3. Observe the errors in the console

**Expected behavior**

The print block in the strict connection checker demo should not have a shadow string block attached, so this error can never occur.

**Additional context**

N/A

### PR DESCRIPTION
**Pull request by mikebarr24**
fixes #1451
I think this is what you were asking for. If not, happy to take feedback.😀

### CODE REVIEW

File: plugins/strict-connection-checker/src/toolboxCategories.js [hunk 1]
```diff
@@ -0,0 +1,849 @@
...
+              'shadow': {
+                'type': 'text',
+                'fields': {
+                  'TEXT': '',
+                },
+              },
+            },
+          },
+        },
+
```
Comment (BeksOmega) [inline plugins/strict-connection-checker/src/toolboxCategories.js]: nit: Remove empty newline please, here and throughout the file =)

File: plugins/strict-connection-checker/src/toolboxCategories.js [hunk 2]
```diff
@@ -0,0 +1,849 @@
...
+                  'TEXT': ',',
+                },
+              },
+            },
+          },
+        },
+        {
+          'type': 'lists_sort',
+          'kind': 'block',
+
```
Comment (BeksOmega) [inline plugins/strict-connection-checker/src/toolboxCategories.js]: nit: remove empty newline

File: plugins/strict-connection-checker/src/toolboxCategories.js [hunk 3]
```diff
@@ -0,0 +1,849 @@
...
+          },
+          'inputs': {
+            'VALUE': {
+              'block': {
+                'type': 'variables_get',
+                'fields': {
+                  'VAR': {
+                    'name': 'text',
+                  },
+                },
```
Comment (BeksOmega) [inline plugins/strict-connection-checker/src/toolboxCategories.js]: It looks like the blocks that include variable blocks are broken as well :/ Could you remove these too?

File: plugins/strict-connection-checker/src/toolboxCategories.js [hunk 4]
```diff
@@ -0,0 +1,849 @@
...
+        {
+          'type': 'text_print',
+          'kind': 'block',
+        },
+        {
+          'type': 'text_prompt_ext',
+          'kind': 'block',
+          'fields': {
+            'TYPE': 'TEXT',
+          },
```
Comment (BeksOmega) [inline plugins/strict-connection-checker/src/toolboxCategories.js]: Thanks for catching this one too!

File: plugins/strict-connection-checker/src/toolboxCategories.js [hunk 5]
```diff
@@ -0,0 +1,849 @@
...
+          'inputs': {
+            'VALUE': {
+              'block': {
+                'type': 'variables_get',
+                'fields': {
+                  'VAR': {
+                    'name': 'list',
+                  },
+                },
+              },
```
Comment (BeksOmega) [inline plugins/strict-connection-checker/src/toolboxCategories.js]: Similarly, all of the lists blocks need their variable sub-blocks removed as well :/

**Review by BeksOmega — COMMENTED** [review]
Thanks for this PR! The fix looks perfect :D But I found some other blocks that were broken as well :/ If you could fix them here that would be wonderful and incredibly appreciated! But if you're busy, this can also be merged after fixing the formatting nits =)

Thank you again for your contribution!

**Comment by mikebarr24** [comment]
Ok I think that's everything. I did a squash merge from upstream into this commit. Hope that hasn't messed things up. First time I've done this to try to avoid an extra merge commit appearing.

**Comment by BeksOmega** [comment]
Hello! Unfortunately I think the squash and merge did mess up your git history a bit :/ Can you run the following commands in your terminal?

```
git checkout master
git pull upstream master
git push origin master
git checkout fix/print_block_strict_connection
git rebase master
git push --force origin fix/print_block_strict_connection
```

Note that you should replace `upstream` with whatever your name for your `google/blockly-samples` remote is, and `origin` with whatever your name for your `mikebarr24/blockly-samples` remote is. You can check your remotes with `git remote -v`.

The `git rebase` should remove the extraneous changes from your latest commit =)

After that I'll be able to take another look and hopefully get this merged!

**Comment by mikebarr24** [comment]
OK I'm now in git hell of my own making 😂
This I don't think is right but I'll figure it out.

**Comment by mikebarr24** [comment]
OK I think this might be it. It's working and passing lint. 
running
`git diff master --name-only`
is showing only the two files I've changed. mikebarr24/blockly-samples master is up to date with google/blockly-samples master.

**Review by BeksOmega — APPROVED** [review]
Sweet looks wonderful! Thank you again for the work on this!

**Pull request merged**